requirements-dev.txt

```diff
@@ -63,6 +63,13 @@ clickclick==20.10.2
     # via
     #   -r requirements.txt
     #   connexion
+colorama==0.4.6
+    # via
+    #   -r requirements.txt
+    #   bandit
+    #   build
+    #   click
+    #   pytest
```

Comment:
these requirements here seem a bit strange, did we mean to add these?

Reply:
Will remove from the changes